### ISSUE TITLE
Stuck launching the program

### ISSUE THREAD

**Issue by Dethkiller15** [BetterDiscord/Installer#307]
**Describe the bug**
I run the .exe on windows 10 and then after showing some resemblance to a normal window the whole thing just becomes a black window. Wait 5 min and nothing changes. Have tried relaunching 6 times and nothing changes from previous attempts.

**Screenshots**
Screenshot shows the window as I am writing this. The program was launched before typing this up.
![image](https://user-images.githubusercontent.com/49108931/207413540-dde7692c-a95b-4b8d-8163-dd6aea6a50c6.png)

**System Information:**
 - OS: Windows 10 home
- Discord Version: Stable

**Comment by fowlis** [comment]
Join the [support server](https://discord.gg/0Tmfo5ZbORCRqbAd) and use [this message link](https://discord.com/channels/86004744966914048/1029513068629205012/1052118051841114172) for solutions.

**Comment by aaaddd4747** [comment]
i am experiencing the same issue, running Microsoft Windows [Version 10.0.19045.2251]

**Comment by Matt0040** [comment]
I had the same problem, but it seems like it just takes an unreasonable amount of time to launch. Try just being patient even for 5 minutes. 

edit: also anti-virus of course

**Comment by FocuzJS** [comment]
Same issue, worked on my other install but on my fresh w11 install its just hung up on the splash screen like the OP

**Comment by Japanese-Schoolgirl** [comment]
I had an outdated version of DiscordPTB on my computer which did not have the "modules" folder. Judging by the error in the installer's console, that's why I had the same problem with blank installer. Updating all versions of Discord clients solve this problem for me and the BD installer starts ran as it should. So it is possible that your problem can be solved in the same way

**Comment by cryocaustik** [comment]
Ran into the same issue - checked task manager and found a few Discord processes still running; killed all those processes and then Discord started up without issue.

**Comment by aaaddd4747** [comment]
> I had the same problem, but it seems like it just takes an unreasonable amount of time to launch. Try just being patient even for 5 minutes.

i left it open for a good hour and it didnt do a thing. details in task manager and process hacker show that the installer has been completely idle the entire time, highest cpu i ever saw from any of its processes was 0.15% which forces me to assume its stuck in a race condition or some other deadlocked state

**Comment by aaaddd4747** [comment]
> I had an outdated version of DiscordPTB on my computer which did not have the "modules" folder. Judging by the error in the installer's console, that's why I had the same problem with blank installer. Updating all versions of Discord clients solve this problem for me and the BD installer starts ran as it should. So it is possible that your problem can be solved in the same way

out of curiosity i dug around and found i too had an old copy of discord ptb installed, the uninstaller for it had long since broken so i had to go into appdata/roaming and manually remove the folders "discordptb" and "Discord PTB". after doing so the betterdiscord installer worked perfectly.

**Comment by neobenedict** [comment]
> Join the [support server](https://discord.gg/0Tmfo5ZbORCRqbAd) and use [this message link](https://discord.com/channels/86004744966914048/1029513068629205012/1052118051841114172) for solutions.

Windows
0. Try https://zerebos.com/BetterDiscord-Windows.exe

A. Run the installer as administrator

B. Right click the installer > properties, click the unblock button and run again.

C. Press win+R and paste ipconfig /flushdns, then restart pc and run the betterdiscord installation.

D. Try disabling any third-party Antivirus software.

E. Manual install: https://discord.com/channels/86004744966914048/1019725965284360283

F. 7-zip method:
1. Install 7-zip https://7-zip.org/
2. Right click the BetterDiscord-windows.exe, and from the 7zip options, click extract to "BetterDiscord-Windows\"
3. Open the new BetterDiscord-Windows folder and run the BetterDiscord.exe
 
Image
Linux
Open the installer through the terminal with --disable-gpu-sandbox appended.

**Comment by zerebos** [comment]
Duplicate of #302
